### DISCUSSION TITLE
How to round only the top part of the Bar in the BarChart?

### DISCUSSION THREAD

**Discussion by cloud-walker** [reaviz/reaviz#245] [Q&A]
Hi! I'm evaluating reaviz, but don't see in the docs how to make the Bar rounded (but only on the top), I've seen the `rx`, `ry` props, but they work very weird (I'm not a super expert about SVGs).

I feel that I'm missing something trivial 🤔

**Comment by amcdnl** [comment]
Ya, those properties round the enire box. There is no equivalent border radius that I know of. I would need to redo it from a rect to a path which is a decent amount of work.

**Reply by cloud-walker** [reply]
I think design wise is a very common use case, and if the goal of the project is to be open to configuration, I think we should support it 🤔 

I am available to contribute.

Recharts already handle this with a `radius` attribute (which support a number or an array of four numbers as value), and found an [article](https://medium.com/@dennismphil/one-side-rounded-rectangle-using-svg-fb31cf318d90) talking about it, but it speak svg related wizardry that I need to dig into first

**Reply by amcdnl** [reply]
Happy to accept a PR - here is where the magic happens: https://github.com/reaviz/reaviz/blob/master/src/BarChart/BarSeries/Bar.tsx#L567
